[v3] Drag n drop example is not running

### Description

I think this error started showing up after the events refactor

```

etesam@Etesams-MacBook-Air ~/C/w/v/e/drag-n-drop (v3-alpha)> go run main.go
# command-line-arguments
./main.go:40:7: app.customEventProcessor undefined (cannot refer to unexported field customEventProcessor)


```

### To Reproduce

1. `git checkout v3-alpha`
2. `cd /examples/drag-n-drop`
3. `go run main.go`

### Expected behaviour

The example runs as expected without the error

### Screenshots

<img width="748" alt="customEventProcessorBug" src="https://github.com/user-attachments/assets/80be775f-6044-4509-8341-4ad2ade7b3a8">


### Attempted Fixes

I think this can be fixed easily by replacing with `app.EmitEvent`

### System Details

```shell
etesam@Etesams-MacBook-Air ~/C/w/v/e/drag-n-drop (v3-alpha) [1]> wails3 doctor

                                
          Wails Doctor          
                                

                                                                                              
# System
┌────────────────────────────┐
| Name          | MacOS      |
| Version       | 15.0       |
| ID            | 24A5331b   |
| Branding      | MacOS 15.0 |
| Platform      | darwin     |
| Architecture  | arm64      |
| Apple Silicon | true       |
| CPU           | Apple M2   |
| CPU           | Unknown    |
| GPU           | Unknown    |
| Memory        | Unknown    |
└────────────────────────────┘

# Build Environment
┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
| Wails CLI      | v3.0.0-alpha.6                                                                                                            |
| Go Version     | go1.23.1                                                                                                                  |
| Revision       | ef8c886b101ab13258998cbb0cfca36db5b110e4                                                                                  |
| Modified       | false                                                                                                                     |
| -buildmode     | exe                                                                                                                       |
| -compiler      | gc                                                                                                                        |
| CGO_CFLAGS     |                                                                                                                           |
| CGO_CPPFLAGS   |                                                                                                                           |
| CGO_CXXFLAGS   |                                                                                                                           |
| CGO_ENABLED    | 1                                                                                                                         |
| CGO_LDFLAGS    |                                                                                                                           |
| DefaultGODEBUG | asynctimerchan=1,gotypesalias=0,httpservecontentkeepheaders=1,tls3des=1,tlskyber=0,x509keypairleaf=0,x509negativeserial=1 |
| GOARCH         | arm64                                                                                                                     |
| GOARM64        | v8.0                                                                                                                      |
| GOOS           | darwin                                                                                                                    |
| vcs            | git                                                                                                                       |
| vcs.modified   | false                                                                                                                     |
| vcs.revision   | ef8c886b101ab13258998cbb0cfca36db5b110e4                                                                                  |
| vcs.time       | 2024-09-08T22:36:19Z                                                                                                      |
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

```
```


### Additional context

_No response_

--- a/mkdocs-website/docs/en/changelog.md
+++ b/mkdocs-website/docs/en/changelog.md
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix for Events API on Linux by [TheGB0077](https://github.com/TheGB0077) in [#3734](https://github.com/wailsapp/wails/pull/3734)
 
 ### Fixed
+- Fixed bug with usage of customEventProcessor in drag-n-drop example by [etesam913](https://github.com/etesam913) in [#3742](https://github.com/wailsapp/wails/pull/3742)
 - [linux] Fixed linux compile error introduced by IgnoreMouseEvents addition by [atterpac](https://github.com/atterpac) in [#3721](https://github.com/wailsapp/wails/pull/3721)
 - [windows] Fixed syso icon file generation bug by [atterpac](https://github.com/atterpac) in [#3675](https://github.com/wailsapp/wails/pull/3675)
 - [linux] Fix to run natively in wayland incorporated from [#1811](https://github.com/wailsapp/wails/pull/1811) in [#3614](https://github.com/wailsapp/wails/pull/3614) by [@stendler](https://github.com/stendler)

--- a/v3/examples/drag-n-drop/main.go
+++ b/v3/examples/drag-n-drop/main.go
@@ -37,10 +37,7 @@ func main() {
 
 	window.OnWindowEvent(events.Common.WindowFilesDropped, func(event *application.WindowEvent) {
 		files := event.Context().DroppedFiles()
-		app.customEventProcessor.Emit(&application.CustomEvent{
-			Name: "files",
-			Data: files,
-		})
+		app.EmitEvent("files", files)
 		app.Logger.Info("Files Dropped!", "files", files)
 	})
 

